src/main/resources/app.conf

```diff
@@ -60,22 +60,29 @@ app {
   # default indexing configuration
   indexing {
     # maximum number of events taken on each batch
-    batch = 10
-    batch = ${?INDEXING_BATCH}
+    batch-chunk = 10
```

Comment:
batch and chunk mean similar things; either keep the name `batch` or rename to `batch-size`